src/modules/commons/modelActions.ts

```diff
@@ -0,0 +1,27 @@
+import * as api from '../../api'
+import * as type from './types'
+
+const modelActions = {
+	getModel: (modelName: string, id?: number) => dispatch => {
```

Comment:
I suggest we have the path, params, query, ... here so it is more generic

Use: `getModel('post', '/post', 1)